app/controllers/auth.php

```diff
@@ -433,8 +435,8 @@ function ($provider, $success, $failure) use ($response, $request, $project) {
     ->desc('OAuth Callback')
     ->label('error', __DIR__.'/../views/general/error.phtml')
     ->label('scope', 'auth')
-    ->label('abuse-limit', 50)
-    ->label('abuse-key', 'ip:{ip}')
+    // ->label('abuse-limit', 100)
```

Comment:
You can completely remove this 2 commented lines, no need for abuse check here as the route only do a redirect anyway.